doc/sphinx/docs/python/python_docs.rst

```diff
@@ -1,4 +1,4 @@
-.. python
+.. python_docs
 
```

Comment:
Just FYI, the syntax in sphinx with a leading ``..`` just means a comment. I think we introduced it by mistake initially, what we really wanted is the syntax ``.. _python_docs:`` instead, which, if placed before a section, generates an anchor called ``python_docs`` to which you can link from other part of the documentation.

We can leave these harmless comments around for this PR (unless you are in a holy doc crusader mood :) ). Eventually, we should get rid of them and replace them with the above syntax - but only where really needed. For instance, if we want to link to a C++ class, we don't need this anchor, as we can link directly to the C++ class via breathe/sphinx.